src/k-provider/ovp/external-captions-builder.js

```diff
@@ -1,5 +1,7 @@
 // @flow
 
+import OVPProviderParser from './provider-parser';
```

Comment:
this imports the entire class, if this is a util move it to a standalone export and import just that